lang/ftidy/Portfile

```diff
@@ -14,7 +15,7 @@ master_sites        ${homepage}
 distname            tidy72sc
 use_zip             yes
 extract.mkdir       yes
-build.args          tidy.for -o ftidy
+build.args          -fdiagnostics-color=always tidy.for -o ftidy
```

Comment:
Why `-fdiagnostics-color=always`? Doesn't that mean that color codes would always be used, even when output is to a logfile? Why would we want that?

Reply:
Never thought about that situation.  I forgot the exact details, but there were situations were the default was set to never. So, I would set it to auto. Do you agree?